EASI.postman_collection.json

```diff
@@ -1,9 +1,9 @@
 {
 	"info": {
```

Comment:
Two potentially useful additions to Postman:
* In the existing "SystemIntake Get (Feedback)" query - add `step` under `actions` as a field to query.
* Either in that query or in a separate query, add the `lcidExpirationChange` field (and all the previous/new) fields under it) under `actions`.

Reply:
sounds good, I updated it.